### ISSUE TITLE
Enter needed during installation process without asking for it?!

### ISSUE THREAD

**Issue by MxNl** [r-lib/rig#117]
Hello,

just installed and used rig for the first time. So please forgive me if I got something wrong :-).

During the installation process I had to press `enter` once, without any hint or question that was asking for it.
I just noticed, because no progress was happening for a few minutes. Maybe I didn't get it right or it was coincidence that after pressing `enter` the process started printing messages again.

Here is the last output before I pressed `enter`
```
[INFO] >   linux-image-4.15.0-65-generic linux-modules-4.15.0-65-generic
[INFO] >   linux-modules-extra-4.15.0-65-generic
[INFO] > Use 'sudo apt autoremove' to remove them.
[INFO] > The following additional packages will be installed:
[INFO] >   libopenblas-dev
[INFO] > The following NEW packages will be installed:
[INFO] >   libopenblas-dev r-4.2.1
[INFO] > 0 upgraded, 2 newly installed, 0 to remove and 343 not upgraded.
[INFO] > Need to get 3.860 kB/63,5 MB of archives.
[INFO] > After this operation, 150 MB of additional disk space will be used.
```

I am on Linux XXX 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

Thanks a lot!

**Comment by gaborcsardi** [comment]
It was most probably a coincidence, but you can run the same command again to make sure. If `apt-get` wants an ENTER, then it prints something like
```
Do you want to continue? [Y/n]
```

**Comment by gaborcsardi** [comment]
Actually, I have just seen this on Ubuntu in a root shell.

**Comment by rstub** [comment]
While setting up a new machine I saw this as well. It looks like the "Do you want to continue? [Y/n]" is echoed only after one presses return:
```
[...]
[INFO] > Need to get 25.8 MB/89.0 MB of archives.
[INFO] > After this operation, 242 MB of additional disk space will be used.
  
[INFO] > Do you want to continue? [Y/n] Get:1 http://apt.pop-os.org/ubuntu jammy/main amd64 bzip2-doc all 1.0.8-5build1 [500 kB]
[INFO] > Get:2 http://apt.pop-os.org/ubuntu jammy/main amd64 libgfortran5 amd64 12-20220319-1ubuntu1 [883 kB]
[...]
```

The empty line above is where I hit return.

**Comment by gaborcsardi** [comment]
Seemingly this only appears from a root shell, when there is no sudo. `apt` thinks that it is running interactively. There is probably a way to make it silent and do the default thing.

**Comment by gaborcsardi** [comment]
Closed in 58e5230
